### ISSUE TITLE
Not all buttons  are Buttons and they don't indicate disability outside color in Widget Builder.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#789]
**Describe the bug**
Buttons don't indicate disability outside color in Design. They are displayed on screen but are not functional.

**To Reproduce**
1. Go to Widget Builder (http://localhost:9992/cm/app/?view=widgetbuilder&site=test&path=%2FSites%2Ftest%2F&pathType=page)
2. The General, Content, Resource and Display buttons are not  buttons and don't indicate disability outside color.

**Expected behavior**
Disability should be indicated and it should be buttons.
